src/main/java/com/bunq/sdk/security/SecurityUtils.java

```diff
@@ -141,6 +146,22 @@
   private static final String DELIMITER_METHOD_PATH = " ";
   private static final String DELIMITER_HEADER_NAME_AND_VALUE = ": ";
 
+  /**
+   * Regex constants.
+   */
+  private static final String REGEX_FOR_LOWERCASE_HEADERS = "(-[a-z])";
+
+  /**
+   * The index of the first item in an array.
+   */
+  private static final int INDEX_FIRST = 0;
+
+  /**
+   * Substring constants.
+   */
+  private static final int SUBSTRING_BEGIN_INDEX = 0;
```

Comment:
Constants can have better names, have no clue what this is about. If we would add a second substring with different params it would be really confusing.